src/task2/api/routsHandlers.js

```diff
@@ -0,0 +1,15 @@
+import { get } from './get';
```

Comment:
I would suggest combine all this handlers in scope of one folder, `user`, `person`, etc. since current approach will not work in case if we will need to add additional API entities.

Reply:
Done